utils.py

```diff
@@ -226,6 +226,29 @@ def getCountDBObjects(dbHost, dbName, login, password):
 		tablesCount[table[2:]] = getCountTable(dbHost, dbName, login, password, table)
 	return tablesCount
 
+def getTableColumnNames(dbHost, dbName, login, password, table):
+	connect = psycopg2.connect(host=dbHost, dbname=dbName, user=login, password=password)
```

Comment:
what the point connect every time, it's better connect once at begging of the test, at setUp functions, then pass connection to helper functions like that

Reply:
Made similar to already existing funcs